src/GitVersionCore/IGitVersionComputer.cs

```diff
@@ -0,0 +1,10 @@
+using GitVersion.OutputVariables;
+
+namespace GitVersion
+{
+    public interface IGitVersionComputer
+    {
+        VersionVariables ComputeVersionVariables(Arguments arguments);
+        bool TryGetVersion(string directory, bool noFetch, out VersionVariables versionVariables);
```

Comment:
The `Try` method should probably match the name of the `ComputeVersionVariables` method and thus be named `TryComputeVersionVariables`, no?

Reply:
Changed to `TryComputeVersionVariables`